blockly-core/core/ui/css.js

```diff
@@ -581,7 +581,7 @@ Blockly.Css.CONTENT = [
   '}',
   '#modalContainer {',
   '  position: absolute;',
-  '  top: 40px;',
+  '  top: 30px;',
```

Comment:
The one place we still have to duplicate this value, unless we want to use applab css to modify this blockly-core style (seemed like a bad idea).  This looks like it would affect the position of the modal editor, but actually it appears to just affect the positioning of the "Save and Close" button.

![blockly_modal_change](https://cloud.githubusercontent.com/assets/1615761/8135429/1cda8ffe-10ec-11e5-9469-9fafa50fe551.png)